src/algorithms/ctmrg/simultaneous.jl

```diff
@@ -1,13 +1,20 @@
 """
-    SimultaneousCTMRG(; tol=Defaults.ctmrg_tol, maxiter=Defaults.ctmrg_maxiter,
-                      miniter=Defaults.ctmrg_miniter, verbosity=0,
-                      projector_alg=Defaults.projector_alg,
-                      svd_alg=SVDAdjoint(), trscheme=FixedSpaceTruncation())
+    struct SimultaneousCTMRG
```

Comment:
Is there a reason the kwarg calling signature was removed? I agree it's a bit redundant given the list of keyword arguments below, but I would at least add a line
```julia
SimultaneousCTMRG(; kwargs...)
```
to the signature to make it clear there is a keyword-only call available.